pkg/agent/httpmux.go

```diff
@@ -35,15 +35,21 @@ func (s *HttpMuxServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	path, found := strings.CutPrefix(r.URL.Path, "/")
 	if !found {
 		w.WriteHeader(http.StatusBadRequest)
-		s.logger.Warn(fmt.Sprintf("MUX: invalid request (path doesn't start with '/'): %s", r.URL.Path))
+		s.logger.Warn(
+			"MUX: invalid request (path doesn't start with '/')",
```

Comment:
If we're already using `logger.Named("mux-server")` for this logger, then we can leave out the "MUX:" prefix.